docs/tools.md

```diff
@@ -84,9 +84,9 @@ Flags:
   -metrictank-url string
     	metrictank address (default "http://localhost:6060")
   -only-org-id value
-    	restrict publishing data belonging to org id; 0 means no restriction (may be given multiple times, once per topic, as a comma-separated list)
+    	restrict publishing data belonging to org id; 0 means no restriction (may be given multiple times, once per topic specified in 'metrics-topic', as a comma-separated list)
   -schemas-file string
-    	path to carbon storage-schemas.conf file (default "/etc/gw/storage-schemas.conf")
+    	path to carbon storage-schemas.conf file (default "/etc/metrictank/storage-schemas.conf")
```

Comment:
per this conversation: https://github.com/grafana/metrictank/pull/1627#discussion_r369821884